File: ComInLanDroid/cominlan/src/main/java/com/onballgroup/cominlan/model/packet/IClientPacket.java
package com.onballgroup.cominlan.model.packet;

import com.onballgroup.cominlan.model.Base.IJson;

/**
 * Created by Phong Le on 4/18/2016.
 */
public interface IClientPacket extends IJson {
    String getId();
    String getName();
    ClientPacketType getType();
}

File: ComInLanWindows/ComInLan/Model/Packet/IClientPacket.cs
using ComInLan.Model.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ComInLan.Model.Packet
{
	public interface IClientPacket : IJson
	{
		string Id { get; }
		string Name { get; }
		ClientPacketType Type { get; }
	}
}
